### PR TITLE
adds samples to topology quick search

### PR DESCRIPTION
**Pull request by invincibleJai**
**Fixes**: 
https://issues.redhat.com/browse/ODC-5787

**Solution Description**: 
adds samples to topology quick search

**Screen shots / Gifs for design review**: 

image

Apr-27-2021 14-15-29

**Browser conformance**: 

- [x] Chrome
- [ ] Firefox
- [ ] Safari
- [ ] Edge

### CODE REVIEW

File: frontend/packages/dev-console/src/components/catalog/providers/useBuilderImageSamples.ts [hunk 1]
```diff
@@ -27,10 +30,12 @@ const normalizeBuilderImages = (
     const provider = annotations?.[ANNOTATIONS.providerDisplayName] ?? '';
     const creationTimestamp = imageStream.metadata?.creationTimestamp;
     const href = `/samples/ns/${activeNamespace}/${name}/${imageStreamNS}`;
+    const createLabel = t('devconsole~Create Application');
+    const type = 'Sample';
 
     const item: CatalogItem = {
-      uid,
-      type: 'Sample',
+      uid: `${type}-${uid}`,
```
Comment (rohitkrai03) [inline frontend/packages/dev-console/src/components/catalog/providers/useBuilderImageSamples.ts:37]: Why is this needed? Isn't uid already unique?

**Review by rohitkrai03 — CHANGES_REQUESTED** [review]

File: frontend/packages/dev-console/src/components/catalog/providers/useBuilderImageSamples.ts (hunk 1 above)
Comment (invincibleJai) [reply frontend/packages/dev-console/src/components/catalog/providers/useBuilderImageSamples.ts:37]: nope, even dev catalog has builder images and they share same ID

**Comment by christianvogt** [comment]
/lgtm
/approve
/hold

Tested quick search in topology for existing for different types of contributions from dev-catalog, quick starts, and samples.
Tested stand alone dev-catalog and samples.

/label qe-approved
/label px-approved added on behalf of as per agreement

fyi @Preeticp @pmkovar for docs-approved label
epic already has docs-ack
